index.html

```diff
@@ -127,7 +114,7 @@ <h2>$1200</h2>
             <li>Lorem ipsum dolor.</li>
             <li>Lorem ipsum.</li>
```

Comment:
I want to suggest using well-detailed content instead of dummy text, It makes your code neater and also much more presentable.